public/app/features/search/components/PreviewsSystemRequirements.tsx

```diff
@@ -0,0 +1,74 @@
+import { Alert, useStyles2 } from '@grafana/ui';
+import React, { FC } from 'react';
+import { config } from '@grafana/runtime/src';
+import { css } from '@emotion/css';
+
+export interface Props {
+  showPreviews?: boolean;
+}
+
+const getText = (requiredImageRendererPluginVersion?: string) => {
```

Comment:
Returning text like this makes it difficult to translate, and it would need to be refactored to accommodate that.

Instead could we contain this fully within a JSX component?

e.g.

```typescript
function Message({ requiredImageRendererPluginVersion }) {
  if (requiredImageRendererPluginVersion) {
    return <>You must update the <Link ...>Grafana image renderer plugin</Link> to version ...</>
  }

  return <>You must install the <Link ...>Grafana image renderer plugin</Link> to enable dashboard previews ...</>...</>
}
```

Reply:
makes sense, done!